data/channels.csv

```diff
@@ -667,6 +667,7 @@ AjaraTV.ge,Ajara TV,,,,GE,,,c/GE,kat,,FALSE,,,,https://ajaratv.ge/,https://i.img
 AJKTV.pk,AJK TV,,,Pakistan Television Corporation,PK,,,c/PK,eng,,FALSE,2004-02-05,,,,https://i.imgur.com/Zb9lawv.png
 Ajman.ae,Ajman,,,,AE,,,c/AE,ara,,FALSE,,,,,https://i.imgur.com/rDekCeE.png
 AjmanTV.ae,Ajman TV,,,,AE,,,c/AE,ara,,FALSE,,,,https://www.ajmantv.com/,https://i.imgur.com/hVNIwgE.png
+AJNetNews.in,AJ Net News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://ajnetnews.com/,https://i.imgur.com/tqJqNCY.png

```

Comment:
Image size should not exceed 512x512: https://github.com/iptv-org/database/blob/master/CONTRIBUTING.md#channels

```suggestion
AJNetNews.in,AJ Net News,,,,IN,,,c/IN,mal,news,FALSE,,,,https://ajnetnews.com/,https://i.imgur.com/DT9VB1Q.png
```